services/app-web/src/components/FileUpload/FileItem/FileItem.tsx

```diff
@@ -120,45 +122,127 @@ export const FileItem = ({
         retryItem(item)
     }
 
-    let statusValue = "";
+    let statusValue = ''
     if (isLoading) {
-        statusValue = "uploading";
+        statusValue = 'uploading'
     } else if (isScanning) {
-        statusValue = "scanning for viruses"
-    } else if (hasDuplicateNameError || hasScanningError || hasUploadError || hasUnexpectedError) {
-        statusValue = "error"
+        statusValue = 'scanning for viruses'
+    } else if (
+        hasDuplicateNameError ||
+        hasScanningError ||
+        hasUploadError ||
+        hasUnexpectedError
+    ) {
+        statusValue = 'error'
     }
 
-    return (
-        <>
-            <div className={styles.fileItemText}>
-                <div
-                    role="progressbar"
-                    aria-valuetext={statusValue}
-                    aria-label={`File status`}
-                >
-                    <img
-                        id={item.id}
-                        data-testid="file-input-preview-image"
-                        src={SPACER_GIF}
-                        alt=""
-                        className={imageClasses}
-                    />
+    if (renderMode === 'list') {
+        return (
+            <>
+                <div className={styles.fileItemText}>
+                    <div
+                        role="progressbar"
+                        aria-valuetext={statusValue}
+                        aria-label={`Status of file ${name}`}
+                    >
+                        <img
+                            id={item.id}
+                            data-testid="file-input-preview-image"
+                            src={SPACER_GIF}
+                            alt=""
+                            className={imageClasses}
+                        />
+                    </div>
+                    <span
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            fontSize: 'inherit',
+                        }}
+                    >
+                        <DocumentError
+                            hasDuplicateNameError={hasDuplicateNameError}
+                            hasScanningError={hasScanningError}
+                            hasUploadError={hasUploadError}
+                            hasUnexpectedError={hasUnexpectedError}
+                        />
+                        <>
+                            {(isLoading || isScanning) && (
```

Comment:
I would like to see some further refinement here to share the conditional logic between the two UI states more. 

I have a bunch of ideas I could share if you want to brainstorm together around this - but at a minimum want to see some kind of shared logic. Even if its just making variables for the conditional logic to reduce all the` ||` and `&& `in JSX.